src/structures/interfaces/InteractionResponses.js

```diff
@@ -162,6 +162,7 @@ class InteractionResponses {
    * @returns {Promise<Message|APIMessage>}
    */
   followUp(options) {
+    if(!this.deferred && !this.replied) throw new Error("NO_INTERACTION_TO_FOLLOWUP");
```

Comment:
I think there's no existing error called `NO_INTERACTION_TO_FOLLOWUP`.
```suggestion
    if(!this.deferred && !this.replied) throw new Error("INTERACTION_NOT_REPLIED");
```